.github/config/evergreen-arm-hardfp.json

```diff
@@ -4,11 +4,14 @@
   "on_device_test": {
     "enabled": true,
     "tests": [
-      "evergreen_test",
       "0",
       "1",
       "2",
-      "3"
+      "3",
+      "4",
```

Comment:
Are you sure this is correct ? I thought we intentionally capped at 4 here due to device availability, and also there are diminishing curve speedups from sharding wider - because of the initalization and transport overhead. 

@arjungm should have a look